### PR TITLE
ci: bump node v22

### PR DESCRIPTION
**Pull request by iiio2**
Since node 22 is stable, I have updated node version to 22.

## Summary by CodeRabbit

- **New Features**
	- Updated Node.js version from 20 to 22 across multiple workflows, enhancing compatibility and performance.

- **Bug Fixes**
	- Improved security by removing default permissions for the `GITHUB_TOKEN` in certain workflows.

- **Chores**
	- Adjusted indentation and formatting in workflow configurations for better clarity.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     permissions:
       id-token: write
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 22
```
Comment (danielroe) [inline .github/workflows/release.yml:20]: 🧐

Comment (iiio2) [reply .github/workflows/release.yml]: @danielroe , changed it

**Comment by iiio2** [comment]
> in general we want to run at least some tests on the _lowest_ node version nuxt supports, to avoid unintended regressions (for example, if using new features of node)

Then will I target for node 18, 20 and 22 ?

**Comment by danielroe** [comment]
let's aim to run fixtures on node 18, and everything else on node 22. sound good?

**Review by danielroe — APPROVED** [review]
❤️

**Comment by iiio2** [comment]
Thanks @danielroe .

**Pull request merged**